src/components/SelectionPills/__tests__/SelectionPills.spec.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { StyleRoot } from 'radium'
+import { mount } from 'enzyme'
+import { spy } from 'sinon'
+import SelectionPills from '../SelectionPills'
+
+describe('SelectionPills', () => {
+  const testPills = [
+    {
+      text: 'bananas',
+      id: 'selection-1',
```

Comment:
Are users always required to pass an explicit id for each pill?

Reply:
Currently, because I didn't want collisions and need to have a unique id to reference in the checkbox and label. I will update to use `text` and rely on the user to know to override when presenting duplicate text (for whatever reason).